docs/modules/user-guide/partials/proc_migrating-plug-ins.adoc

```diff
@@ -1,12 +1,10 @@
 [id="proc_migrating-plug-ins_{context}"]
-= Migrating plug-ins
+= Migrating plugins
```

Comment:
Plugins have been removed from the specification (see https://github.com/devfile/api/pull/333). The editor-specific configuration is taking a different approach. @l0rd This is one of the areas that need work on for the migration guide.